promtimer/promtimer.py

```diff
@@ -252,7 +285,63 @@ def main():
     grafana_port = args.grafana_port
     prometheus_base_port = grafana_port + 1
 
-    if not args.cluster:
+    cbbackupmgr_stats_mode = False
+    buckets = None
+
+    if args.stats_archive_path:
+        cbbackupmgr_stats_mode = True
+
+        archive_path = args.stats_archive_path
+        if zipfile.is_zipfile(args.stats_archive_path):
+            # Using tempfile.gettempdir() sometimes causes the Prometheus server to take a long time to start up
+            archive_path = os.path.join(tempfile.gettempdir(), 'stats-archive')
+            if os.path.exists(archive_path):
+                shutil.rmtree(archive_path)
+            os.mkdir(archive_path)
+
+            with zipfile.ZipFile(args.stats_archive_path, 'r') as zipped_archive:
+                zipped_archive.extractall(archive_path)
+
+            archive_path = os.path.join(archive_path, os.path.splitext(os.path.basename(args.stats_archive_path))[0])
+        # Check that stats_archive_path exists and is not an empty dir
+        elif not os.path.isdir(args.stats_archive_path) or \
+            (len(os.listdir(args.stats_archive_path)) == 0):
```

Comment:
No need for the outer brackets here. Small thing but it confused me for a second - I thought you were doing something more complicated than you are